gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/NoUnusedPinCheckTask.java

```diff
@@ -38,7 +38,9 @@ public NoUnusedPinCheckTask(File propsFile) {
 
     @Input
     public final Set<String> getResolvedArtifacts() {
-        return BaselineVersions.getResolvedArtifacts(getProject());
+        return getProject().getAllprojects().stream()
```

Comment:
I am pretty sure that this is infinite recursion as getAllProjects returns the project itself.